packages/js/src/ws/socket.ts

```diff
@@ -34,7 +98,7 @@ export class Socket extends BaseModule {
 
   #notificationReceived = ({ message }: { message: TODO }) => {
     this.#emitter.emit(NOTIFICATION_RECEIVED, {
-      result: new Notification(message),
+      result: new Notification(mapToNotification(message)),
```

Comment:
the WS event emits the message entity which should we map to the Notification class object